revolv/static/project.scss

```diff
@@ -43,6 +43,405 @@
+                font-size: 0.8rem;
+            }
+        }
+
+        .category-holder {
+            display: inline-block;
+            border: 1px solid #888888;
+            width: 15%;
+            height: 30px;
+            margin: 10px 15px inherit 15px;
+
+            .category-label {
+                color: #888888;
+                text-align: center;
+                line-height: 28px;
+            }
+        }
+    }
+
+    .about-text {
+        p {
+            font-weight: 500;
+            font-size: 1.5rem;
+            color: #333333;
+        }
+        margin-bottom: 80px;
+    }
+
+}
+
+.project-updates {
+    .header-text {
+        p {
+            font-weight: 800;
+            color: black;
+            font-size: 1.8rem;
+        }
+    }
+    .gray-bar {
+        background-color: #999999;
+        height: 1px;
+        width: 100%;
+    }
+
+    .update-container {
+        margin-top: 40px;
+        width: 100%;
+        margin-bottom: 40px;
+        display: inline-block;
+        position: relative;
+
+        .date-container {
+            height: 50px;
+            width: 100%;
+            text-align: center;
+            color: $revolv-dark-blue;
+            font-weight: 300;
+            line-height: 0.5;
+
+            .date-year {
+                font-size: 4.5rem;
+                margin-bottom: 0rem;
+                line-height: 4.5rem;
+            }
+
+            .date-month-day {
+                font-size: 1.5rem;
+                line-height: 1.5rem;
+            }
+        }
+
+        .update-text {
+            font-weight: 500;
+            font-size: 1.5rem;
+            color: #333333;
+        }
+    }
+}
+
+.map {
+    margin-top: 40px;
+    margin-bottom: 50px;
+    height: 400px;
+
+    .map-canvas {
+        position: absolute;
+        width: 100%;
+        height: 400px;
+        margin: 0;
+        padding: 0;
+
+        img {
+            max-width: none;
+        }
+    }
+
+    .overlay {
+        background: rgba(0, 0, 0, 0.65);
+        float: left;
+        height: 400px;
+    }
+
+    .map-cover {
+        background: rgba(0,0,0,0);
+        height: 400px;
+        float: right;
+    }
+
+    .text-container {
+        position: absolute;
+        color: white;
+        width: 80%;
+        margin-left: 10%;
+        margin-right: 10%;
+        margin-top: 35px;
+
+        .title-container {
+            p {
+                font-weight: 500;
+                font-size: 1.8rem;
+                line-height: 1.8rem;
+            }
+        }
+
+        .address-container {
+            p {
+                font-weight: 500;
+                font-size: 1.5rem;
+                margin: 0;
+                line-height: 1.5rem;
+            }
+        }
+    }
+
+    .donate-button-header {
+        border: 2px solid white;
+        width: 80%;
+        height: 70px;
+        z-index: 100;
+        margin: 305px 10% 25px 10%;
+        transition: all 0.3s;
```

Comment:
i added in -moz-transform and -webkit-transform because safari needed them for some reason

safari doesn't seem like it needs -moz-transition or -webkit-transition but i'll add them in to be safe